plugins/modules/win_firewall.ps1

```diff
@@ -13,11 +13,15 @@ $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "b
 
 $profiles = Get-AnsibleParam -obj $params -name "profiles" -type "list" -default @("Domain", "Private", "Public")
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -failifempty $true -validateset 'disabled','enabled'
+$inbound_action = Get-AnsibleParam -obj $params -name "inbound_action" -type "str" -default "block" -validateset 'allow','block'
```

Comment:
Should we have a default value, right now if people don't define it then the module will block the connection whereas before it just ignored the value. Sounds like we don't want a default and to only do the check if it's defined. We should also add `not_configured` as an option as the underlying cmdlet supports this. You would need a switch to convert the snake_case to the PascalCase the cmdlets expect but that shouldn't be too hard.